File: server/objects.py
class MileStone:
    def __init__ (self, id, name, dueDate, completed):
        self.id = id
        self.name = name
        self.dueDate = dueDate
        self.completed = completed

class Project:
    def __init__ (self, id, name, dueDate, mileStones, completed, hidden):
        self.id = id
        self.name = name
        self.dueDate = dueDate
        self.mileStones = mileStones
        self.completed = completed
        self.hidden = hidden

class User:
    def __init__ (self, id, name, projects):
        self.id = id
        self.name = name
        self.projects = projects

File: src/Objects.js
// Project Object
export class Project {
    constructor(id, name, mileStones, dueDate, completed) {
        this.id = id;
        this.name = name;
        this.mileStones = mileStones; // Array of Milestones
        this.dueDate = dueDate; // date
        this.completed = completed; //boolean
    }
}

// Milestone Object
export class MileStone {
    constructor(id, name, dueDate, completed) {
        this.id = id;
        this.name = name;
        this.dueDate = dueDate; // date
        this.completed = completed; // boolean
    }
}

// User Object
export class User {
    constructor(id, name, projects) {
        this.id;
        this.name = name;
        this.projects = projects; // Array of Projects
    }
}